auth/localauth.go

```diff
@@ -40,6 +40,9 @@ func defaultAuth() Client {
 
 // Instance returns initialized instance of auth client
 func Instance() Client {
+	if _std == nil {
```

Comment:
Good point. removing this check actually. I actually don't need default instance here, and I don't want lock in the request path.